docs/pages/usage.mdx

```diff
@@ -4,6 +4,12 @@ sidebar_label: Usage
 ---
 
 ## Install Chart
+
+```bash
+helm repo add devspace https://charts.devspace.sh
+helm repo update
+```
+
```

Comment:
I'm not sure if this step is necessary. Since the `--repo` flag is used in the commands below, they likely don't need this command.

If we want to add this to the docs, then the commands below might be updated to use the repo in the chart name instead of `--repo`:
```sh
helm install -n my-namespace my-component devspace/component-chart
```

There's also a question of whether we want to use `https://charts.loft.sh` or `https://charts.devspace.sh`. I _think_ they point to the same chart repo.

Reply:
@lizardruss sorry for dropping the ball on this one, adding it to todo.

Reply:
Yes, looks like they point to the same repo, I will use loft.sh

```bash
➜ for domain in charts.loft.sh charts.devspace.sh; do
  echo -n "$domain -> "
  dig +short $domain | tr '\n' ' '
  echo ""
done
charts.loft.sh -> 34.121.62.77
charts.devspace.sh -> charts.loft.sh. 34.121.62.77
```